### PR TITLE
Trigger on comment

### LINKED ISSUE #16: Trigger on PR comment

### PR DESCRIPTION
**Pull request by isacarvid**

### CODE REVIEW

**Comment by ayubatif** [comment]
test comment

**Comment by isacarvid** [comment]
hello

File: .github/workflows/python.yml [hunk 1]
```diff
@@ -1,6 +1,11 @@
 name: Notifier
 on:
-  push:
+  pull_request:
+    types: [opened]
+  issue_comment:
```
Comment (ayubatif) [inline .github/workflows/python.yml]: hello world

**Review by ayubatif — CHANGES_REQUESTED** [review]
needs work

**Comment by isacarvid** [comment]
hello

**Review by ayubatif — COMMENTED** [review]
nothing

**Review by ayubatif — APPROVED** [review]
LGTM

**Review by ayubatif — CHANGES_REQUESTED** [review]
nvm

**Comment by isacarvid** [comment]
testing

File: .github/workflows/python.yml [hunk 2]
```diff
@@ -1,7 +1,9 @@
 name: Notifier
 on:
   pull_request_review_comment:
-    types: [created, deleted]
+    types: [created, deleted, edited]
```
Comment (ayubatif) [inline .github/workflows/python.yml]: bare review comment single

File: .github/workflows/python.yml [hunk 3]
```diff
@@ -16,14 +19,13 @@ jobs:
       - name: Get README.md
         run: |
           git fetch --quiet
-          changed=$(git diff origin/master HEAD --name-only)
```
Comment (ayubatif) [inline .github/workflows/python.yml:19]: why

**Review by ayubatif — CHANGES_REQUESTED** [review]

**Review by ayubatif — APPROVED** [review]

File: .github/workflows/python.yml [hunk 4]
```diff
@@ -1,7 +1,5 @@
 name: Notifier
 on:
-  pull_request_review_comment:
-    types: [created, deleted, edited]
```
Comment (ayubatif) [inline .github/workflows/python.yml]: this doesnt work

**Review by ayubatif — CHANGES_REQUESTED** [review]
pick another topic

**Review by ayubatif — APPROVED** [review]
nvm LGTM

**Pull request merged**